code/Backend/requirements.txt

```diff
@@ -0,0 +1,4 @@
+pip install Flask
```

Comment:
Might want to check the format for requirements.txt file, I believe it is just flas-cors, psycogpg2-binary etc. so we don't need pip install. The reason to use this file is so that other people executing/using/testing this code can invoke pip install requirements.txt (so the pip install part should already be there in the command line), just make sure this is the correct format.

Reply:
That's right, I do remember us speaking about this process in our meeting 7 about the requirements.txt file as an alternative to docker containers.

Reply:
I updated based on Jay's comment